.github/workflows/ci.yml

```diff
@@ -294,7 +345,12 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'pnpm'
       - name: pnpm install
-        run: pnpm install --frozen-lockfile  
+        uses: nick-fields/retry@v2.8.3
+        with:
+          max_attempts: 10
+          timeout_minutes: 15
+          retry_on: error
+          command: pnpm install --frozen-lockfile
```

Comment:
Is it possible to reuse this job configuration?

Reply:
I don't really know how. Maybe let's do it once I get CI green again.